Makefile

```diff
@@ -1,8 +1,9 @@
 ROOT_DIR := .
-SERVER_EXECUTEBLE=$(ROOT_DIR)/pkg/cmd/server/server
+SERVER_EXECUTEBLE := "$(ROOT_DIR)/pkg/cmd/server/server"
 
-OVN_KUBERNETES_ROOT := \
-	~/work/src/github.com/ovn-org/ovn-kubernetes
+ifndef OVN_KUBERNETES_ROOT
+override OVN_KUBERNETES_ROOT := "$(ROOT_DIR)/../../ovn-org/ovn-kubernetes"
+endif
```

Comment:
use 
```
OVN_KUBERNETERS_ROOT ?= $(ROOT_DIR)/../../ovn-org/ovn-kubernetes
```

Reply:
it doesn't work. I checked it

Reply:
I want to allow to set OVN_KUBERNETERS_ROOT as an environment variable, plus there is a default value

Reply:
```
vagrant@vagrant:~/projects/openvswitch$ unset OVN_KUBERNETES_ROOT
vagrant@vagrant:~/projects/openvswitch$ make
echo xxx/../../ovn-org/ovn-kubernetes
xxx/../../ovn-org/ovn-kubernetes
vagrant@vagrant:~/projects/openvswitch$ cat Makefile
ROOT_DIR=xxx
OVN_KUBERNETES_ROOT ?= $(ROOT_DIR)/../../ovn-org/ovn-kubernetes

all:
        echo ${OVN_KUBERNETES_ROOT}

vagrant@vagrant:~/projects/openvswitch$
```

Reply:
ok, and now try to do the same when you set OVN_KUBERNETES_ROOT

Reply:
ok